### ISSUE TITLE
aria label is not announced when using the ARROW keys in browse mode in FireFox 

### ISSUE THREAD

**Issue by rnickelson** [nvaccess/nvda#6361]
Open the attached file in Firefox navigate with the arrow keys notice that NVDA does not announce First Name.
